packages/metablocks/src/accounts/utils.ts

```diff
@@ -56,10 +56,11 @@ const setBlockMetadata = async (
   return metadata;
 };
 
-const getAllAccountInfo = async <T>(
+const getAllAccountInfo = async (
   idlAccountName: string,
   program: Program,
-  layout: BufferLayout.Structure
+  v1Layout: Layout,
```

Comment:
This needs to change to array, loop through all provided layouts until deserialised correctly. Will do this in the next interation